auth-web/src/components/auth/common/ContactUsTooltip.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-tooltip top max-width="450px" light content-class="tooltip" v-model="showTip">
+    <template v-slot:activator="{ attrs }">
+      <span :attrs="attrs" @click="showTip = !showTip" class="tooltip-text">Contact Us</span>
+    </template>
+    <v-card class="tooltip-content">
+      <h3 class="mb-3">Don't have a Passcode for your Cooperative Association?</h3>
+      <p>If you have no received your Access Letter from BC Registries, or have lost your Passcode, please contact us
+        at:</p>
+      <section>
+        <p class="mb-0">Toll Free: 1-877-526-1526</p>
+        <p class="mb-0">Phone: 250-387-7848</p>
+        <p>Email: bcregistries@gov.bc.ca</p>
+      </section>
+      <v-spacer></v-spacer>
+      <section>
+        <p class="mb-0">Hours of Operation:</p>
+        <p>Monday to Friday, 8:30am - 4:30pm PST</p>
+      </section>
+      <a href="">Frequently Asked Questions</a>
```

Comment:
Can't go MVP with this.

Reply:
For sure, same as above. Should i remove the tooltip template until we have all the info for implementation.

Reply:
Ah, um...

Either way there is tech debt (either to add the file or update it later). I suggest leaving in, possibly with a big comment block at the top that it's incomplete and not used yet. You may avoid future merge issues if you do this instead of keeping this file on another branch to merge later.

Reply:
Done and done, in next commit!